__tests__/service/anonymousMessageRepo.test.ts

```diff
@@ -41,7 +41,7 @@ describe('TESTING anonymousMessageRepo', () => {
 
       // Assertion
       expect(result).toEqual({});
-      expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat/activityId');
+      expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat-v2/activityId');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from '/feeds/feed-chat/activityId' to '/feeds/feed-chat-v2/activityId' seems appropriate as it aligns with the PR's purpose of updating the endpoint URLs. However, ensure that the new endpoint '/feeds/feed-chat-v2/activityId' is correctly configured and tested on the server side.
```diff
- expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat/activityId');
+ expect(anonymousApi.get).toBeCalledWith('/feeds/feed-chat-v2/activityId');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->